.github/workflows/lint-and-test.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       - name: Run cargo test (proof primitives - Dory) (std feature only - i.e. not using blitzar)
         run: cargo test proof_primitive::dory::dory_compute_commitments_test --no-default-features --features="std"
       - name: Run hello_world example
-        run: cargo run --example hello_world --features="blitzar test"
+        run: cargo run --example hello_world --no-default-features --features="test"
```

Comment:
Two changes:
1) We should use rayon when not using `blitzar`.
2) We should run with both `--no-default-features --features="test rayon"` and ` --features="test"`.

Reply:
so we need to run two tests one with `test rayon` and another with only `test`. Am I correct ?

Reply:
The one with "test rayon" needs to have --no-default features in order to turn off blitzar.